metricbeat/docs/running-on-docker.asciidoc

```diff
@@ -19,7 +19,7 @@ docker run \
   --volume=/sys/fs/cgroup:/hostfs/sys/fs/cgroup:ro \ <2>
   --volume=/:/hostfs:ro \ <3>
   --net=host <4>
```

Comment:
This line needs a trailing backslash for line continuation. Should be: `--net=host \ <4>`